src/planner/operator/alter_table.rs

```diff
@@ -0,0 +1,19 @@
+use crate::catalog::{ColumnCatalog, TableName};
+
+#[derive(Debug, PartialEq, Clone)]
+pub enum AlterTableOperator {
+    AddColumn(AddColumn),
```

Comment:
Convert these enumerations into their corresponding Operators so that they can correspond better when writing execution operators, otherwise they will be coupled together. Maybe the Alert-related Operators can be placed in a directory to categorize them.